app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -467,6 +467,8 @@ def resource_groups
         :name    => resource[:name]
       )
     end
+  rescue IBMCloudSdkCore::ApiException
+    nil
```

Comment:
I think it'd be good to be more specific about which errors we're skipping.

```suggestion
    rescue IBMCloudSdkCore::ApiException => err
    raise unless err.message.match?(/whatever the exception is/)
    
    nil
```

And include a comment about why we're doing this?